src/site/content/en/blog/declarative-link-capturing/index.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: post
+title: Choose how in-scope links open your PWA with Declarative Link Capturing
+subhead: |
+  Declarative Link Capturing is a proposal for a Web Application Manifest property called
+  `"capture_links"` that lets developers determine declaratively what should happen when the browser
+  is asked to navigate to a URL that is within the application's navigation scope, from a context
+  outside of the navigation scope.
+authors:
+  - thomassteiner
+date: 2021-05-12
+description: |
+  Declarative Link Capturing is a proposal for a Web Application Manifest property called
+  "capture_links" that lets developers determine declaratively what should happen when the browser
+  is asked to navigate to a URL that is within the application's navigation scope, from a context
+  outside of the navigation scope.
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/ynRFiUuv3sccX4qUAAPw.jpg
+alt: A hand and several hanging chains, symbolizing links.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - progressive-web-apps
+  - web-app-manifest
+origin_trial:
+  url: https://developer.chrome.com/origintrials/#/view_trial/4285175045443026945
+---
+
+{% Aside %} Declarative Link Capturing is part of the
+[capabilities project](https://web.dev/fugu-status/) and is currently in development. This post will
+be updated as the implementation progresses. {% endAside %}
+
+## What is Declarative Link Capturing? {: #what }
+
+Clicking links on the web can sometimes be a bit of an adventure. For example, clicking a
```

Comment:
I might not phrase this opening sentence like this. "a bit of an adventure" sounds negative, and what we're essentially proposing is an API to make that click more "adventurous". Could we find a more positive way to introduce this topic?

Reply:
"Positively surprising experience"